src/main/java/com/sgio/yieldseeker/builder/ApartmentBuilder.java

```diff
@@ -0,0 +1,134 @@
+package com.sgio.yieldseeker.builder;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.sgio.yieldseeker.enumerations.Convenience;
+import com.sgio.yieldseeker.enumerations.DPE;
+import com.sgio.yieldseeker.enumerations.ExtraSpace;
+import com.sgio.yieldseeker.enumerations.Heating;
+import com.sgio.yieldseeker.model.Apartment;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ApartmentBuilder {
+    private String city;
+    private Integer postalCode;
+    private Float surfaceArea;
+    private DPE dpe;
+    private Heating heating;
+    private Boolean parking;
+    private List<ExtraSpace> extraSpaces = new ArrayList<>();
+    private List<Convenience> conveniences = new ArrayList<>();
+    private List<String> photoLinks = new ArrayList<>();
+    private Float score = 0f;
+
+    public ApartmentBuilder from(JsonObject jsonDatas) {
```

Comment:
```suggestion
    public ApartmentBuilder from(final JsonObject jsonDatas) {
```
Add final for params as much as possible if not always